src/main/java/com/microsoft/azure/servicebus/jms/ServiceBusJmsConnection.java

```diff
@@ -3,15 +3,15 @@
 
 package com.microsoft.azure.servicebus.jms;
 
-import javax.jms.Connection;
-import javax.jms.ConnectionConsumer;
-import javax.jms.ConnectionMetaData;
-import javax.jms.Destination;
-import javax.jms.ExceptionListener;
-import javax.jms.JMSException;
-import javax.jms.ServerSessionPool;
-import javax.jms.Session;
-import javax.jms.Topic;
+import jakarta.jms.Connection;
+import jakarta.jms.ConnectionConsumer;
+import jakarta.jms.ConnectionMetaData;
+import jakarta.jms.Destination;
+import jakarta.jms.ExceptionListener;
+import jakarta.jms.JMSException;
+import jakarta.jms.ServerSessionPool;
+import jakarta.jms.Session;
+import jakarta.jms.Topic;
```

Comment:
Just adding a comment here but that is really for the PR description:

1.  Remove this in PR description:

During testing I found out Jndi bug introduced by autodelete on entities

2. For title, lets use something like:
Update to Jakarta messaging Client and other dependencies.